acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
I placed the certs in a temp directory. What's the problem now.

Btw, the old code was supposed to be removing the certs directory when stopping the cluster. Was it not working properly for you?